Dockerfile

```diff
@@ -5,20 +5,29 @@ ARG DOCKER_GID=497
 USER root
 
 ENV DOCKER_VERSION "18.03.1-ce"
+
 RUN apk add --no-cache curl shadow \
     && curl -L -o /tmp/docker-$DOCKER_VERSION.tgz https://download.docker.com/linux/static/stable/x86_64/docker-$DOCKER_VERSION.tgz \
     && tar -xz -C /tmp -f /tmp/docker-$DOCKER_VERSION.tgz \
     && mv /tmp/docker/docker /usr/bin \
     && rm -rf /tmp/docker-$DOCKER_VERSION /tmp/docker \
-    && apk del curl
+    && apk del curl \
```

Comment:
We should just not remove curl here, so that it doesn't have to be re-added below.